egi/api_resources/gamelisting/checkin.py

```diff
@@ -46,11 +46,20 @@
 )
 
 post_parser.add_argument(
-    'connected_player_count', dest='connected_player_count', type=int,
+    'connected_account_count', dest='connected_player_count', type=int,
     location='form', required=False,
 )
 post_parser.add_argument(
-    'total_player_count', dest='total_player_count', type=int,
+    'total_account_count', dest='total_player_count', type=int,
+    location='form', required=False,
+)
+
+post_parser.add_argument(
+    'connected_player_count', dest='fallback_connected_player_count', type=int,
+    location='form', required=False,
+)
+post_parser.add_argument(
+    'total_player_count', dest='fallback_total_player_count', type=int,
```

Comment:
Similarly, I'd match this up as well as `total_player_count`

Reply:
There you have it! I've made those requested changes - the model fields match Evennia 0.7, and the parser accepts both *_player_count and *_account_count, preferring the account keys.

Reply:
You'll also have to change these templates: https://github.com/gtaylor/evennia-game-index/tree/master/egi/templates